src/vs/workbench/parts/debug/electron-browser/replViewer.ts

```diff
@@ -7,13 +7,10 @@ import * as nls from 'vs/nls';
 import { TPromise } from 'vs/base/common/winjs.base';
```

Comment:
@isidorn You're right, did not see that for-loop when refactored. I've moved LinkDetector instantiation  to the constructor.